server/src/packages/bbls/bbls.controller.ts

```diff
@@ -34,17 +34,22 @@ export class BblsController {
   create(@Body() body) {
     const allowedAttrs = pick(body, BBL_ATTRS);
 
-    if (!body.pas_form || !body.project) throw new HttpException(
-      'Missing pas_form or project ids',
-      HttpStatus.UNPROCESSABLE_ENTITY,
-    );
+    if (
+      !(body.pas_form || body.landuse_form)
+      || !body.project
```

Comment:
Hmmm, this is interesting.

So, we have many different types of forms, and will have more. It seems like this is making sure its dependencies — various relationships — are included. This is sort of a validation step.

I don't think it's worth doing right now but maybe there is another pipe of some sort, before the deserializer, that validates the input, based on some validation schema.

Reply:
Thanks for this, I think it's a good idea! Will create a task for it